geometry_msgs/Vector3 p1
geometry_msgs/Vector3 p2
geometry_msgs/Vector3 p3
